docs/hpc/08_ood/open_on_demand.md

```diff
@@ -0,0 +1,184 @@
+# Open OnDemand (OOD) with Conda/Singularity
+
+[Open OnDemand](https://ood.hpc.nyu.edu/) is a tool that allows users to launch Graphical User Interfaces (GUIs) based applications are accessible without modifying your HPC environment. You can log into the Open OnDemand interface at [https://ood.hpc.nyu.edu](https://ood.hpc.nyu.edu). Once logged in, select the **Interactive Apps** menu, select the desired application, and submit the job based on required resources and options. 
+
+## OOD + Singularity + conda
+This page describes how to use your Singularity with conda environment in Open OnDemand (OOD) GUI at Greene. 
+
+### Log Into Greene via the Terminal
+The following commands must be run from the terminal. Information on accessing via the terminal can be found at the [Connecting to the HPC page](../02_connecting_to_hpc/01_connecting_to_hpc.md). 
+
+### Preinstallation Warning
+If you have initialized Conda in your base environment (your prompt on Greene may show something like `(base) [NETID@log-1 ~]$)` then you must first comment out or remove this portion of your ~/.bashrc file:
+
+```
```

Comment:
Specify language, `sh` to enable syntax highlighting. https://docusaurus.io/docs/markdown-features/code-blocks#syntax-highlighting

Reply:
Cool, that is nicer.  I'll make that change.  Thanks for pointing it out!